libraries/botbuilder-dialogs-adaptive/src/entityAssignmentComparer.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { AdaptiveEvents } from './adaptiveEvents';
+import { EntityAssignment } from './entityAssignment';
+
+export class EntityAssignmentComparer {
+    private static eventPreference: string[] = [
```

Comment:
Generally, `tsc` can infer the type based on the type of the initial value. In those cases, prefer to omit the explicit type.